src/ol/geom/geometry.js

```diff
@@ -132,6 +132,7 @@ ol.geom.Geometry.prototype.getClosestPoint = function(point, opt_closestPoint) {
 /**
  * @param {ol.Coordinate} coordinate Coordinate.
  * @return {boolean} Contains coordinate.
+ * @api
```

Comment:
Right, I'm only questioning the use of the word "contains".  It may be more appropriate to call this "intersects".